aiotg/bot.py

```diff
@@ -519,6 +520,10 @@ def __del__(self):
 
     def _process_message(self, message):
         chat = Chat.from_message(self, message)
+        chat_id = message['chat']['id']
+
+        if chat_id not in self._conversation:
+            self._conversation[chat_id] = asyncio.Queue(maxsize=3)
```

Comment:
Not everyone needs conversations support so it's better to add some flag to enable/disable this feature. It can be set on bot initialisation or implicitly when using response().

Maybe not a flag, but change it to something like:
```
conversation = self._conversations.get(chat_id)
```

(conversation queue should be created in response())